diff --git a/index.tsx b/index.tsx
index 6464ae12194d6ed49d2e1a3fe201490178c49cb2..b41e2b1144ef821850c5c5e05d42eb5d444c179c 100644
--- a/index.tsx
+++ b/index.tsx
@@ -58,8 +58,8 @@ interface Props {
 
     onBackdropPress?: (...args: any) => any,
     pickerButtonOnPress: (item: CountryItem) => any,
-    itemTemplate?: (props: ItemTemplateProps) => JSX.Element,
-    ListHeaderComponent?: (props: ListHeaderComponentProps) => JSX.Element,
+    itemTemplate?: (props: ItemTemplateProps) => React.ReactElement,
+    ListHeaderComponent?: (props: ListHeaderComponentProps) => React.ReactElement,
     onRequestClose?: (...args: any) => any,
 
     lang: string,
@@ -339,8 +339,8 @@ interface CountryListProps {
     popularCountries?: string[],
     showOnly?: string[],
 
-    ListHeaderComponent?: (props: ListHeaderComponentProps) => JSX.Element,
-    itemTemplate?: (props: ItemTemplateProps) => JSX.Element,
+    ListHeaderComponent?: (props: ListHeaderComponentProps) => React.ReactElement,
+    itemTemplate?: (props: ItemTemplateProps) => React.ReactElement,
     pickerButtonOnPress: (item: CountryItem) => any,
 
     style?: Style,
